sample-jira-setup.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
Why there are two json files `jira/jira-v1.json` and `sample-jira-setup.json`?  the second one does not configure the projects endpoint